src/main/com/microsoft/azure/documentdb/changefeedprocessor/services/ChangeFeedJob.java

```diff
@@ -10,18 +32,15 @@
 import com.microsoft.azure.documentdb.changefeedprocessor.internal.ChangeFeedThreadFactory;
 import com.microsoft.azure.documentdb.changefeedprocessor.internal.StatusCode;
 import com.microsoft.azure.documentdb.changefeedprocessor.internal.SubStatusCode;
-import lombok.Getter;
 
 import java.util.List;
 import java.util.concurrent.*;
 import java.util.logging.Logger;
```

Comment:
we should use slf4j not jul logger. but again probably not in the scope of this PR. just for your note.

Reply:
Not in this scope.